api-testing/functional/tests/cypress/integration/01-institution/patch-institutions-institution.ts

```diff
@@ -0,0 +1,32 @@
+describe("Institution api", () => {
+  it("PATCH - Update institution details", () => {
+    cy.fixture("token").then((data) => {
+      cy.request({
+        method: Cypress.env("patch"),
+        url: Cypress.env("update_institution_url"),
+        auth: { bearer: data.access_token },
```

Comment:
I am not following this. Is the access token not retrieved from the keylock directly?

Reply:
https://github.com/bcgov/SIMS/pull/1237#discussion_r883758001